etc/ci/check_no_panic.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 #
-# Make sure listed files do not contain "unwrap"
+# Make sure listed files do not contain "panic"
 set -o errexit
 set -o nounset
 set -o pipefail
 
 cd $(git rev-parse --show-toplevel) # cd into repo root so make sure paths works in any case
 
-# files that should not contain "unwrap"
+# files that should not contain "panic"
```

Comment:
I actually think we can get rid of this comment, the first one makes this redundant.